deploy/README.md

```diff
@@ -361,31 +375,6 @@ Note: When executing the helm upgrade to avoid the error below is need add the a
 
 If all goes well, you should now have your custom metrics piping into Sumo Logic.
 
-### Events
```

Comment:
I see you made a new PR for moving this to the other file, in the future feel free to keep that in this same PR 🙂